### ISSUE TITLE
Trailblazed symbols

### ISSUE THREAD

**Issue by Hades1503** [osmandapp/OsmAnd#11380]
# 🚀 feature request

### Describe the solution you'd like

 Have you considered any alternative solutions or workarounds?

**Comment by miramikes** [comment]
Hello,
I as author of this proposal would have couple of suggestions how to render these tags.
I wanted to ask same, but someone was quicker ;)
Following link shows rendering of poles besides winter routes.
https://en.mapy.cz/zimni?x=15.6739326&y=50.7022212&z=16&source=bs&id=1985564114
I think to render trailblazed=* as offset tiny marks beside path/tracks/roads is good idea
trailblazed=yes ... little circle
trailblazed=no ... not rendered
trailblazed=poles ... little cross x (poles sometimes having cross on top for better visibility)
trailblazed=cairns ... little triangle (reminding shape of cairn)
trailblazed=symbols ... little rhombus
... not sure about color :( ... red? ... and size 

there is also additional tags about visibility of blazing based mainly on distance between blazing 
trailblazed:visibility=excellent
trailblazed:visibility=good
trailblazed:visibility=intermediate
trailblazed:visibility=bad
trailblazed:visibility=horrible
trailblazed:visibility=no

I would suggest make three density patterns:
quite dense for:
trailblazed:visibility=excellent
trailblazed:visibility=good
normal density for:
trailblazed:visibility=intermediate
trailblazed:visibility=bad
and less dense for:
trailblazed:visibility=horrible
trailblazed:visibility=no ... this tag is defined as there are still some marks even very rare.

If tag trailblazed:visibility=* is missing ( what will be very probably in most cases) I would stay with middle density of rendering

best regards

mira

**Comment by Hades1503** [comment]
hello mira,

you had a very good idea!
I have already started tagging
trailblazed = symbols
+
osmc:symbol = :: blue_dot 
(if it's a blue dot on the trail)

https://taginfo.openstreetmap.org/tags/trailblazed=symbols

I think that with the osm: symbol we can better detail the shape and color of the markup

however, in osmand the osmc: symbol are widely spaced and do not represent all the marks if there is more than one route that takes the same path. someone has already brought up this display problem.

**Comment by miramikes** [comment]
> hello mira,
> 
> you had a very good idea!
> I have already started tagging
> trailblazed = symbols
> +
> osmc:symbol = :: blue_dot
> (if it's a blue dot on the trail)
> 
> https://taginfo.openstreetmap.org/tags/trailblazed=symbols
> 
> I think that with the osm: symbol we can better detail the shape and color of the markup
> 
Yes that would be probably good idea, but there should be also rendering for case when trailblazed=symbols doesn't have  osmc:symbol
>
> however, in osmand the osmc: symbol are widely spaced and do not represent all the marks if there is more than one route that takes the same path. someone has already brought up this display problem.
>
Yes ... my answer on this was ... Tag it by a stronger symbol if there is one or just simply by trailblazed=yes.

**Comment by xmd5a2** [comment]
OsmAnd rendering engines doesn't support offsets for icons along path. So this is not possible at that moment.

**Comment by Hades1503** [comment]
otherwise the display of symbols when you click on the path. this implies that all the trails are clickable to display information other than those already displayed on the screen.

trail_visibility
the visibility of the path is only represented in osmand if it is bad. good, intermediate, excellent, there is no difference displayed.


assisted_trail, safety_rope, rungs, is not shown. this is very important information for hiking. this could avoid getting into difficulty

**Comment by miramikes** [comment]
I opened trailblazed=* feature request to not mix too topics together ... this topic is by it's name about trailblazed=symbols only

**Comment by miramikes** [comment]
> 

Hello,
I know nothing about "rendering engines" which OsmAnd using. They are developed in OsmAnd project or they are some sort of 3rd party source?
I am just interested if such a feature like offset icons, offset lines and clickable lines will be be possible in foreseeable future or it is rather long therm run?
best regards
mira
